deploy/truenas_ws.sh

```diff
@@ -175,7 +175,7 @@ truenas_ws_deploy() {
   then
     _err "Error calling system.ready:"
     _err "$_ws_response"
-    exit $_ws_re
+    exit $_ws_ret
```

Comment:
please help remove all the `exit`.   use `return` instead.

Reply:
Done